lib/scarpe/wv/image.rb

```diff
@@ -1,14 +1,16 @@
 # frozen_string_literal: true
 
-require "base64"
-require "uri"
+# require "base64"
+# require "uri"
+require_relative "../base64"
```

Comment:
Remove rather than commenting out the lines above (plus same require request as above.)